pkg/build/builder/docker.go

```diff
@@ -312,6 +312,21 @@ func (d *DockerBuilder) dockerBuild(dir string, tag string, secrets []buildapi.S
 	}
 	opts.NetworkMode = network
 	if len(resolvConfHostPath) != 0 {
+		/*
+			label, err := getSELinuxLabel()
+			if err != nil {
+				return fmt.Errorf("could not retrieve selinux label due to error: %v", err)
+			}
+			if len(label) != 0 {
+				opts.SecurityOpt = []string{label}
+			}
+		*/
+		cmd := exec.Command("chcon", "system_u:object_r:svirt_sandbox_file_t:s0", "/etc/resolv.conf")
```

Comment:
Why is the commented out code still here?  Just testing?  Also, what are all the ways this can fail?

Reply:
dead code at this point, i'll remove it.

all the ways this can fail are that if we can't fix the permissions on the file, then the container will either fail to start (can't mount the resolv.conf) or won't have proper DNS behavior.

but hopefully if the chcon fails, we'll get an error here and abort the build before we get to that point.